docs/index.rst

```diff
@@ -30,6 +30,7 @@ partitions and selects subsets of data transparently.
 To get started, have a look at our :doc:`guide/getting_started` guide,
 head to the description of the :doc:`spec/index` or head straight to the API documentation :doc:`api`.
 
+To get started with Cube Functionality, have a look at our :doc:`guide/cube/getting_started` guide,
```

Comment:
At this point the user doesn't have a clue what "Cube Functionality" is so I would remove this sentence or at least work in some kind of hint what this includes

Reply:
I agree.. Will remove the line.